### PR TITLE
MFTF: Sharing Wishlist with more than allowed Text Length Limit

### LINKED ISSUE #28969: [Issue] MFTF: Sharing Wishlist with more than allowed Text Length Limit
This issue is automatically created based on existing pull request: magento/magento2#28812: MFTF: Sharing Wishlist with more than allowed Text Length Limit

---------
### Preconditions (*)
Magento 2.4-develop
This PR contains a test for sharing customer's wishlist with more than allowed Text Length Limit

### Steps to reproduce (*)
1 - Set maximum allowed Text Length Limit for sharing
2 - Login as a customer
3 - Add product into wishlist
4 - Share wishlist with more than maximum allowed Text Length Limit
5 - Perform Assertions

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
This PR contains a test for sharing customer's wishlist with more than allowed Text Length Limit

Steps to reproduce:

1 - Set maximum allowed Text Length Limit for sharing
2 - Login as a customer
3 - Add product into wishlist
4 - Share wishlist with more than maximum allowed Text Length Limit
5 - Perform Assertions

### Resolved issues:
1. [x] resolves magento/magento2#28969: MFTF: Sharing Wishlist with more than allowed Text Length Limit

### CODE REVIEW

**Comment by dmytro-ch** [comment]
@magento run all tests

File: app/code/Magento/Wishlist/Test/Mftf/Test/StorefrontShareWishlistWithMoreThanMaximumAllowedTextLengthLimitTest.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            <group value="configuration"/>
+        </annotations>
+        <before>
+            <magentoCLI command="config:set wishlist/email/text_limit 10" stepKey="changeTextLengthLimit"/>
+            <magentoCLI command="cache:clean config" stepKey="cleanCache"/>
+            <createData entity="SimpleSubCategory" stepKey="createCategory"/>
+            <createData entity="SimpleProduct" stepKey="createProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+            <magentoCLI command="cron:run --group=index" stepKey="runCronIndexer"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Wishlist/Test/Mftf/Test/StorefrontShareWishlistWithMoreThanMaximumAllowedTextLengthLimitTest.xml:28]: We have magentoCron command available 👍

**Review by lbajsarowicz — APPROVED** [review]
The contribution is fine, but after verifying error you can add steps to verify if after amends it works properly. Why? Sometimes the validators get stuck with some js / session data issues

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7717](https://jira.corp.magento.com/browse/ENGCOM-7717) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-7717](https://jira.corp.magento.com/browse/ENGCOM-7717) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
@magento create issue

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**